.github/workflows/reusable_dockerfile_pipeline.yml

```diff
@@ -75,6 +85,11 @@ jobs:
           echo "SHORT_SHA=`echo ${GITHUB_SHA} | cut -c1-8`" >> $GITHUB_ENV
           # yamllint disable
           echo "IMAGE_NAME=$(echo ${{ github.repository }} | tr '[:upper:]' '[:lower:]')" >> $GITHUB_ENV
+          # here we validate if we have specified a different package name in
+          # the inputs, if so, we change the package to it.
+          if [[ ${{ inputs.packageName }} != ${{ github.repository}} ]];then
+            echo "IMAGE_NAME=$(echo ${{ github.repository_owner }}/${{ inputs.packageName }} | tr '[:upper:]' '[:lower:]')" >> $GITHUB_ENV
```

Comment:
Code duplication: Is it possible to put that in one action and use the output from both actions? This way, we only need to change one place to add functionality.

Reply:
I couldn't share these variables between jobs very clear... 
I saw the `outputs` but looks a little bit tricky... like [this](https://stackoverflow.com/a/63485275), not sure if it worth it